proto/internal/dispatch/v1/dispatch.proto

```diff
@@ -123,6 +124,26 @@ message DispatchLookupResources2Response {
   Cursor after_response_cursor = 3;
 }
 
+message DispatchLookupResources3Request {
+  ResolverMeta metadata = 1 [(validate.rules).message.required = true];
+
+  core.v1.RelationReference resource_relation = 2 [(validate.rules).message.required = true];
+  core.v1.RelationReference subject_relation = 3 [(validate.rules).message.required = true];
+  repeated string subject_ids = 4;
+
+  core.v1.ObjectAndRelation terminal_subject = 5 [(validate.rules).message.required = true];
```

Comment:
This name is a little confusing to me at first glance -- specifically how it's different than `subject_ids` (I suspect it's on redispatch but a comment might make things clearer)